<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="my_label" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>451</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mouse_movement">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>    Mouse Movement</string>
    </property>
   </widget>
   <widget class="QLabel" name="mouse_pressed">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>180</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse Pressed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(451 X 451)</string>
    </property>
   </widget>
   <widget class="Line" name="x_axis">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>225</y>
      <width>450</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 85, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="y_axis">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>2</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_axes">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Axes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setgridbutton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Grid</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="gridspinbox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>540</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setpoint1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>point1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setpoint2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>point2</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>82</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="DDAline">
       <property name="text">
        <string>DDA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="time_dda">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>330</y>
      <width>87</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="bresenhamLine">
       <property name="text">
        <string>Bresenham</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="time_Bresenhum">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>410</y>
      <width>160</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="radius_circle"/>
     </item>
     <item>
      <widget class="QPushButton" name="Centre_circle">
       <property name="text">
        <string>choose centre</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>450</y>
      <width>101</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="bresenham">
       <property name="text">
        <string>bresenham</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="circle_bresenham_time">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>450</y>
      <width>111</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="midpoint_circle">
       <property name="text">
        <string>midpoint </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="circle_midpoint_time">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>450</y>
      <width>111</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="circle_polar">
       <property name="text">
        <string>polar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="circle_polar_time">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>my_label</class>
   <extends>QLabel</extends>
   <header location="global">my_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
